docker-compose.override.yml

```diff
@@ -151,14 +166,12 @@ services:
   webshoppingagg:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
-      - urls__basket=http://basket-api
-      - urls__catalog=http://catalog-api
-      - urls__identity=http://identity-api
+      - IdentityUrl=http://identity-api
+      - IdentityUrlExternal=http://${ESHOP_EXTERNAL_DNS_NAME_OR_IP}:5105
+      - SeqServerUrl=http://seq
       - BasketUrlHC=http://basket-api/hc
       - CatalogUrlHC=http://catalog-api/hc
       - IdentityUrlHC=http://identity-api/hc
-      - PaymentUrlHC=http://payment-api/hc
```

Comment:
Yes, the web shopping aggregator doesn't need to include the payment API's health status because it doesn't call the payment API. Thanks for checking!

Reply:
OK, trying to update my fork with your changes and noticed that one. 😄